sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -601,14 +601,21 @@ def getBandInformation(self, statistics=True, dataset=None, **kwargs):
             self._bandInfo = infoSet
         return self._bandInfo
 
+    @property
+    def geospatial(self):
```

Comment:
It may be good to put this property on the base tile source class and return False by default

Reply:
It can just be a constant on the base class (see https://github.com/girder/large_image/blob/patch/geospatial-attr/large_image/tilesource/base.py#L47).

Reply:
Ah right 🤦🏻‍♂️